src/test/java/snippets/SnippetsTests.java

```diff
@@ -322,13 +324,18 @@ public void nthElement_return_nth_element_from_end_when_n_is_less_than_0() throw
 
     @Test
     public void pick_should_pick_key_pairs_corresponding_to_keys() throws Exception {
-        Map<String, Integer> obj = new HashMap<>();
-        obj.put("a", 1);
-        obj.put("b", 2);
-        obj.put("c", 3);
+        
+        Map<String, Integer> obj = new LinkedHashMap<>();    
+        obj.put("a", 1);    
```

Comment:
How do these three `put` lines differ from before? Why would they appear in the diff? There may be again some white-space changes (maybe spaces vs. TABs?).